sdks/python/apache_beam/runners/dataflow/dataflow_runner.py

```diff
@@ -378,6 +379,23 @@ def run_Flatten(self, transform_node):
           PropertyNames.ENCODING: step.encoding,
           PropertyNames.OUTPUT_NAME: PropertyNames.OUT}])
 
+  def apply_WriteToBigQuery(self, transform, pcoll):
```

Comment:
As I read the code, this is using the native bigquery sink for batch on direct runner. Should we be doing the same for the dataflow runner (i.e. using the native sink for batch)?